README.md

```diff
@@ -1,20 +1,57 @@
 # Eventing performance tests
 
+## Prerequisites:
+- Have an OCP cluster ready (it will be scaled up to 15 workers)
```

Comment:
Worker count can be changed, 15 is the default that is documented in ./bin/run.sh` so I wouldn't duplicate this info here as well.


```suggestion
- Access to an OCP cluster
```